assets/bingo/lang/de_de.json

```
@@ -1,17 +1,364 @@
 {
+	"%1074992263$s%1$s": "%2$s",
+	"bingo.technical.detect_multiplayer": "DoNotTranslateThis",
+
+	"bingo.resourcepack_check.sign.line1": "",
+	"bingo.resourcepack_check.sign.line2": "Klick",
+	"bingo.resourcepack_check.sign.line3": "Mich",
+	"bingo.resourcepack_check.sign.line4": "",
+	"bingo.resourcepack_check.go_to_lobby.part1": "Das Ressourcenpaket wurde erfolgreich aktiviert.",
+	"bingo.resourcepack_check.go_to_lobby.part2": "%s um zur Lobby zu gehen.",
+	"bingo.resourcepack_check.go_to_lobby.click": "Hier klicken",
+
+	"bingo.game.go_to_lobby": "%s um zur Lobby zurückzukehren.",
+	"bingo.game.go_to_lobby.hint": "Hinweis: Du kannst jederzeit mit dem Befehl %s zur Lobby zurückkehren",
+	"bingo.game.go_to_lobby.click": "Hier klicken",
+
+	"bingo.preferences.title": "Spieler Einstellungen ändern",
+	"bingo.preferences.description": "Dieses Menü erlaubt die Konfiguration deiner Einstellungen.",
+	"bingo.preferences.back": "Vorheriges Menu",
+	"bingo.preferences.previous": "Zurück",
+	"bingo.preferences.next": "Weiter",
```

```suggestion
	"bingo.preferences.back": "Zurück",
	"bingo.preferences.previous": "Vorh. Seite",
	"bingo.preferences.next": "Nächste Seite",
```
I think it's better to keep the general "back" shorter